dev/com.ibm.ws.cloudant_fat/publish/files/couchdb-ssl/couchdb-config/testcontainers_config.ini

```diff
@@ -1,4 +1,4 @@
-; Disable clustering and sharding since this DB will only ever be used
```

Comment:
Shard is actually the correct verb here.  Database sharding is a way to distribute data between database instances to help with performance.

Reply:
Great catch!  I have put back the original wording and learned something in the process.  Thanks!